test/phoenix/integration/websocket_channels_test.exs

```diff
@@ -11,15 +11,6 @@ defmodule Phoenix.Integration.WebSocketChannelsTest do
   @moduletag :capture_log
```

Comment:
Changes to this file are a large diff, but amount to nesting existing tests inside a comprehension over bandit and cowboy servers (in order to test both the existing `Cowboy2Handler` call path, and the new `Sock` based one.